app/error_messages.py

```diff
@@ -40,3 +40,6 @@
     "Variants have more than one question type for block."
 )
 PREVIEW_WITHOUT_INTRODUCTION_BLOCK = "No introduction block found. Introduction block is mandatory when using the preview questions feature."
+ANSWER_REFERENCED_BEFORE_ADDED = (
```

Comment:
Not sure about `before_added` here again, as per the list PR, maybe it needs to before it exists or before answered 🤔

Reply:
This one is hard to get right for me.. "..BEFORE_EXISTS"?